wye/profiles/views.py

```diff
@@ -97,3 +100,38 @@ def dispatch(self, *args, **kwargs):
             return super(ProfileEditView, self).dispatch(*args, **kwargs)
         else:
             raise PermissionDenied
+
+
+class ContactFormView(FormView):
+    form_class = ContactUsForm
+    template_name = 'contact.html'
+    success_url = '/thankyou'
+
+    def form_valid(self, form):
+
+        email_context = Context({
+            'contact_name': form.cleaned_data['name'],
+            'contact_email': form.cleaned_data['email'],
+            'comments': form.cleaned_data['comments'],
+            'conatct_number': form.cleaned_data['contact_number'],
+            'feedback_type': form.cleaned_data['feedback_type']
+            })
+
+        subject = "PythonExpress Feedback by %s" % (form.cleaned_data['name'])
+        text_body = loader.get_template(
+            'email_messages/contactus/message.txt').render(email_context)
+        email_body = loader.get_template(
+            'email_messages/contactus/message.html').render(email_context)
+
+        try:
+            regional_lead = models.Profile.objects.filter(
```

Comment:
1) Can we extend for user type ='admin' also .
2) We need to even send confirmation email to user(from form we get email id) saying we will work on there feedback and get back to them soon